### PR TITLE
[W5][T11-2]Matthias Lum

### PR DESCRIPTION
**Pull request by matthiaslum**
Added a single junk command.

### CODE REVIEW

**Comment by liguanlong** [comment]
Although this new feature is not very useful, the added code follows the OOP style in a good manner. I believe if given more time, you will be able to carry out a much more valuable modification. Keep it up!

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -47,6 +47,14 @@ Format: `help`
 Help is also shown if you enter an incorrect command e.g. `abcd`
 ====
 
+== Adding Junk : `junk`
+
+Format: `junk`
+
+[TIP]
```
Comment (leerachel) [inline docs/UserGuide.adoc:54]: This seems like an introduction of your command, maybe it would be better to put it above the "Format: " line? Sort of like an explanation of your command, instead of it being a [TIP]. Also, try to follow the "standard" user guide format so that your teammates and you can produce a uniform writing style for a more user-friendly user guide. E.g. Instead of "A single line of junk is printed...", maybe rephrase to "Prints a single line of junk..." as you can see from the other commands' introductions "Adds a person.. ", "Deletes a person...".

File: src/seedu/addressbook/commands/JunkCommand.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package seedu.addressbook.commands;
+
+/**
+ * Shows help instructions.
+ */
+public class JunkCommand extends Command {
+
+    public static final String COMMAND_WORD = "junk";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": THIS IS A LINE OF UNNECESSARY JUNK. YOU JUST WASTED 1 MIN OF YOUR LIFE =)\n"
```
Comment (leerachel) [inline src/seedu/addressbook/commands/JunkCommand.java:10]: Take note of indentation! Always try to keep the code neat and readable :) Try to avoid lines longer than 110 chars.

File: src/seedu/addressbook/parser/Parser.java [hunk 3]
```diff
@@ -96,7 +97,8 @@ public Command parseCommand(String userInput) {
 
         case ExitCommand.COMMAND_WORD:
             return new ExitCommand();
-
+        case JunkCommand.COMMAND_WORD:
```
Comment (leerachel) [inline src/seedu/addressbook/parser/Parser.java:100]: Try inserting a blank line before and after this case to make it neater!

**Review by leerachel — COMMENTED** [review]
Nice attempt, hope to see more creative commands from you! Please close this PR after reading comments.

**Comment by WuPeiHsuan** [comment]
It's a nice try~~~

**Comment by shawn-t** [comment]
Hey matthias, good enhancement attempt! 👍 For the parser file, is there supposed to be a line of "return new JunkCommand();" after your "case JunkCommand.COMMAND_WORD:" @ line 100?

**Comment by sickerin** [comment]
@matthiaslum Very cool command, it’s as useful as your Tiny, i.e complete junk.
